frmts/opjlike/jp2opjlikedataset.cpp

```diff
@@ -0,0 +1,3745 @@
+/******************************************************************************
+ *
+ * Project:  JPEG2000 driver based on OpenJPEG or Grok library
+ * Purpose:  JPEG2000 driver based on OpenJPEG or Grok library
+ * Author:   Aaron Boxer, <boxerab at protonmail dot com>
```

Comment:
I wouldn't mind being kept as one of the co-authors:
```suggestion
 * Authors:   Even Rouault, <even dot rouault at spatialys dot com>
 *            Aaron Boxer, <boxerab at protonmail dot com>
```

Reply:
Good idea, done.